components/hab/src/cli/hab/bldr.rs

```diff
@@ -77,6 +83,9 @@ pub struct JobStatus {
 }
 
 /// Cancel a build job group and any in-progress builds
+///
+/// NOTICE: Public Builder Build Functions are no longer supported. Please reach
+/// out to your account team if you were using these features.
 #[derive(ConfigOpt, StructOpt)]
 #[structopt(name = "cancel", no_version)]
 pub struct JobCancel {
```

Comment:
We might want to get rid of the options to this as well. same as `JobStatus` above.